File: java/ContactsApp/app/src/main/java/com/egehanasal/contactsapp/MainActivity.java
package com.egehanasal.contactsapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;

import android.os.Bundle;
import android.view.View;

import com.egehanasal.contactsapp.databinding.ActivityMainBinding;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;
    static ArrayList <Contact> contactArrayList;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);

        contactArrayList = new ArrayList<>();

        Contact contact1 = new Contact("name1", "surname1", "12345678", R.drawable.colloseum);
        Contact contact2 = new Contact("name2", "surname2", "98765432", R.drawable.eiffel);
        Contact contact3 = new Contact("name3", "surname3", "13244657", R.drawable.londonbridge);
        Contact contact4 = new Contact("name4", "surname4", "47585123", R.drawable.pisa);

        contactArrayList.add(contact1);
        contactArrayList.add(contact2);
        contactArrayList.add(contact3);
        contactArrayList.add(contact4);

        binding.recyclerView.setLayoutManager(new LinearLayoutManager(this));
        ContactAdapter contactAdapter = new ContactAdapter(contactArrayList);
        binding.recyclerView.setAdapter(contactAdapter);
    }
}

File: kotlin/ContactsApp/app/src/main/java/com/egehanasal/contactsapp/MainActivity.kt
package com.egehanasal.contactsapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Menu
import android.view.MenuInflater
import android.view.MenuItem
import android.widget.LinearLayout
import androidx.recyclerview.widget.LinearLayoutManager
import com.egehanasal.contactsapp.databinding.ActivityDetailsBinding
import com.egehanasal.contactsapp.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding
    private lateinit var contactList: ArrayList<Contact>

    override fun onCreate(savedInstanceState: Bundle?) {
        supportActionBar?.hide()
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        contactList = ArrayList<Contact>()

        val contact1 = Contact("happy", "face", 23, R.drawable.face)
        val contact2 = Contact("normal", "face2", 22, R.drawable.facetwo)
        val contact3 = Contact("sad", "face3", 23, R.drawable.facethree)

        contactList.add(contact1)
        contactList.add(contact2)
        contactList.add(contact3)

        binding.recyclerViewTextView.layoutManager = LinearLayoutManager(this)
        val contactAdapter = ContactAdapter(contactList)
        binding.recyclerViewTextView.adapter = contactAdapter
    }
}